### DISCUSSION TITLE
3.18.8 release?

### DISCUSSION THREAD

**Discussion by lfcnassif** [sepinf-inc/IPED#703] [Q&A]
All devs,

I'm thinking about a new bug fix release because of the first 3 fixes at https://github.com/sepinf-inc/IPED/blob/3.18.x/ReleaseNotes.txt

I also thought about including #416 (very modular) and maybe #415. Does someone else want/think it's important to include another ticket?

Nassif

**Comment by lfcnassif** [comment]
@tc-wleite I'm trying to cherry pick other small improvements without merge conflicts (or easy to solve). What do you think about including #206?

**Reply by wladimirleite** [reply]
I think it is a good idea. There is a performance gain that can help in cases with many items to be OCRed, and I believe the risk of introducing problems is minimal.

**Comment by lfcnassif** [comment]
A final question here. I'm in doubt about including #558 because of its initialization time (#643 causes merge conflicts and I don't want to rewrite and test it again). Should #558 be included in 3.18.8?

**Reply by lfcnassif** [reply]
It wasn't included.
